README.md

```diff
@@ -520,6 +520,14 @@ model_config:
     type: # linear | dynamic
     factor: # float
 
+# optional overrides to the bnb 4bit quantization configuration
+# https://huggingface.co/docs/transformers/main/main_classes/quantization#transformers.BitsAndBytesConfig
+bnb_config_kwargs:
+  # These are default values
+  # llm_int8_has_fp16_weight: false
+  # bnb_4bit_quant_type: nf4
+  # bnb_4bit_use_double_quant: true
```

Comment:
Perhaps having these three lines be uncommented is clearer?  Since the top level is uncommented?  What do you think?
```suggestion
  llm_int8_has_fp16_weight: false
  bnb_4bit_quant_type: nf4
  bnb_4bit_use_double_quant: true
```

Reply:
The reason is the rest of the config documentation has the key/value pairs uncommented including optional ones?

Reply:
That makes sense! I applied your change